### PR TITLE
[flux-core v0.13] Add top level README

### PR DESCRIPTION
**Pull request by cmoussa1**
This PR adds a top-level README with titles and descriptions of each example as well as hyperlinks to each directory on branch `master`.

### CODE REVIEW

**Comment by cmoussa1** [comment]
Rebased and force-pushed to get up to date with `master` after PR #46.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+**_7. [Using Flux Job Status and Control API](https://github.com/flux-framework/flux-workflow-examples/tree/master/example7)_**
+
+Submit job bundles and wait until all jobs complete
+
+**_8. [Simple KVS Python Binding Example](https://github.com/flux-framework/flux-workflow-examples/tree/master/example9)_**
+
+Use KVS Python interfaces to store user data into KVS
+
+**_9. [A Data Conduit Strategy](https://github.com/flux-framework/flux-workflow-examples/tree/master/example10)_**
```
Comment (SteVwonder) [inline README.md]:
```suggestion
**_9. [A Data Conduit Strategy](https://github.com/flux-framework/flux-workflow-examples/tree/master/example9)_**
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+**_6. [Hierarchical Launching](https://github.com/flux-framework/flux-workflow-examples/tree/master/example6)_**
+
+Launch a large number of sleep 0 jobs
+
+**_7. [Using Flux Job Status and Control API](https://github.com/flux-framework/flux-workflow-examples/tree/master/example7)_**
+
+Submit job bundles and wait until all jobs complete
+
+**_8. [Simple KVS Python Binding Example](https://github.com/flux-framework/flux-workflow-examples/tree/master/example9)_**
```
Comment (SteVwonder) [inline README.md]:
```suggestion
**_8. [Simple KVS Python Binding Example](https://github.com/flux-framework/flux-workflow-examples/tree/master/example8)_**
```

**Comment by cmoussa1** [comment]
🤦‍♂️ I forgot to make those changes. Thanks for pointing that out @SteVwonder! Fixed those hyperlinks and force-pushed.

**Comment by cmoussa1** [comment]
Thanks!

**Pull request merged**